From 5548568fc4419ece6e274344fdaaf3932d8a3522 Mon Sep 17 00:00:00 2001
From: Eric Engestrom <eric.engestrom@imgtec.com>
Date: Tue, 25 Apr 2017 11:40:27 +0100
Subject: [PATCH drminfo 2/3] meson: install drminfo & drmtest

`gtktest` had an explicit `install: false`, but it's the other way
around: if unspecified, the file doesn't get installed; you have to
explicitely ask for it with `install: true`.

Signed-off-by: Eric Engestrom <eric.engestrom@imgtec.com>
---
 meson.build | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 4722bf9..8312179 100644
--- a/meson.build
+++ b/meson.build
@@ -24,10 +24,12 @@ gtktest_deps  = [ gtk3_dep,
 
 executable('drminfo',
            sources      : drminfo_srcs,
-           dependencies : drminfo_deps)
+           dependencies : drminfo_deps,
+           install      : true)
 executable('drmtest',
            sources      : drmtest_srcs,
-           dependencies : drmtest_deps)
+           dependencies : drmtest_deps,
+           install      : true)
 executable('gtktest',
            sources      : gtktest_srcs,
            dependencies : gtktest_deps,
-- 
Cheers,
  Eric

